src/telemetry/pings.yaml

```diff
@@ -9,6 +9,19 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/pings/2-0-0
 
+main:
+  description: |
+    A simple ping to send glean events generated by the Mozilla Cuddlefish apps.
+  include_client_id: true
+  send_if_empty: false
+  bugs:
+    - https://bugzilla.mozilla.org/show_bug.cgi?id=1708637
+  data_reviews:
+    - https://bugzilla.mozilla.org/show_bug.cgi?id=1711123
+  notification_emails:
+    - amarchesini@mozilla.com
+    - vpn-telemetry@mozilla.com
```

Comment:
Why was this added?

Reply:
This is coming from src/shared/telemetry/pings.yaml